src/web/nextui/src/app/eval/ResultsTable.tsx

```diff
@@ -209,32 +213,6 @@ function EvalOutputCell({
 
   let text = typeof output.text === 'string' ? output.text : JSON.stringify(output.text);
   let node: React.ReactNode | undefined;
-  let chunks: string[] = [];
-  let hasImageLightbox = false;
-  if (text.startsWith('![')) {
-    const imageUrlRegex = /^!\[.*?\]\((.*?)\)/;
-    const imageUrlMatch = text.match(imageUrlRegex);
-    if (imageUrlMatch) {
-      const url = imageUrlMatch[1];
-      node = (
-        <>
-          <img loading="lazy" src={url} alt={output.prompt} onClick={toggleLightbox} />
-          {lightboxOpen && (
-            <div className="lightbox" onClick={toggleLightbox}>
-              <img src={url} alt={output.prompt} />
-            </div>
-          )}
-        </>
-      );
-      hasImageLightbox = true;
-    }
-  } else if (!output.pass && text.includes('---')) {
-    // TODO(ian): Plumb through failure message instead of parsing it out.
-    chunks = text.split('---');
-    text = chunks.slice(1).join('---');
```

Comment:
We should preserve this "chunks" behavior - it's what parses out and displays failures nicely.

With split:
<img width="541" alt="image" src="https://github.com/promptfoo/promptfoo/assets/310310/0c996956-6fee-435e-a373-c9a1dea56465">

Without split:
<img width="535" alt="image" src="https://github.com/promptfoo/promptfoo/assets/310310/4ba7c38e-59e4-4eea-a260-5431d61f1164">

Reply:
ah right let me fix that!

Reply:
@typpo when you test with the `echo` provider - are you entering that config via the web ui yaml?

or are you defining a yaml file somewhere, running an eval somehow other than the web ui, and only inspecting the results with the web ui?

the reason I ask is because I get a 500 error when attempting to test only via the web ui. I think it has to do with the `IS_RUNNING_LOCALLY` env var?

the error:
```
error StaticGenBailoutError: Page with `dynamic = "error"` couldn't be rendered statically because it used `request.json`
```

which seems to come from: 
https://github.com/promptfoo/promptfoo/blob/fdb296d97523e4c53c304580dfc2ce9268fbbf2e/src/web/nextui/src/app/api/eval/job/route.ts#L15